### ISSUE TITLE
Add docs on how CockroachDB handles  "split-brain" scenarios (network partitions)

### ISSUE THREAD

**Issue by rmloveland** [cockroachdb/docs#12159]
Per [this forum conversation](https://forum.cockroachlabs.com/t/split-brain-scenarios-in-multi-az/4795), the behavior is as follows:

> In a split brain scenario ongoing operations are suspended and
> wait/block until the partition resolves. There are no writes during
> the partition (they are blocked), so we donât need to re-sync. The
> block is only for the minority side, so the majority side can
> proceed. Writes donât get committed until a majority has
> acknowledged them. If the acknowledgement went through, the write is
> ok; if the acknowledgement is blocked, the write is considered
> aborted. Reads that encounter a write thatâs too recent (i.e. weâre
> not sure 100% that itâs not going to abort) need to wait, or retry,
> or push the write forward in time. Ongoing operations resume when
> the partition resolves.

We should probably update the [network partition troubleshooting docs](https://www.cockroachlabs.com/docs/dev/cluster-setup-troubleshooting.html#network-partition) with some of this info.  It could also perhaps link to/from the [bounded staleness reads stuff](https://www.cockroachlabs.com/docs/dev/follower-reads.html#bounded-staleness-reads)

**Comment by taroface** [comment]
Hi @thtruo @mwang1026 , do we consider this more of an Observability or a Disaster Recovery issue?

cc @kathancox

**Comment by mwang1026** [comment]
I actually consider this more OX since OX owns network and behavior in light of network issues. cc @piyush-singh  :)

Happy to take this offline too given that so much of the solution is visibility into partitions (and to avoid a hot potato problem)

for the record, this is "resilience" IMO and no "disaster" (my personal opinion)

**Comment by piyush-singh** [comment]
Agreed, I think this should live with the docs on the network latency matrix or similar where we describe how to identify a network partition. If we want to have a dedicated page on network partitions, we could split that out, but it should still be either Observability or Deployments and Ops (DB Server) in my opinion.
